Example/Frameworks/Database/Logger.swift

```diff
@@ -27,35 +27,133 @@ import UIKit
 import Willow
 
 extension LogLevel {
-    /// Custom log level for SQL log messages with a bitmask of `1 << 8`.
-    public static let sql = LogLevel(rawValue: 0b00000000_00000000_00000001_00000000)
+    // Namespace our custom levels
+    public enum Database {
+        /// Custom log level for SQL log messages with a bitmask of `1 << 8`.
+        public static var sql = LogLevel(rawValue: 0b00000000_00000000_00000001_00000000)
+    }
 }
 
 // MARK: -
 
+// Extend Logger to have `sql` log level functions.
 extension Logger {
-    func sql(_ message: @autoclosure @escaping () -> String) {
-        logMessage(message, with: .sql)
+    func sql(_ message: @autoclosure @escaping () -> Message) {
+        logMessage(message, with: LogLevel.Database.sql)
     }
```

Comment:
We namespace the custom `LogLevel`, but not the convenience functions on `Logger`. Should we? Probably don't need to if the API is internal, but thought I'd bring it up.